ApplicationLibCode/ReservoirDataModel/RigWellResultPoint.h

```diff
@@ -70,6 +71,9 @@ struct RigWellResultPoint
 
     cvf::Vec3d bottomPosition() const;
 
+    std::string cellIjk() const;
+    void        setIjk( std::string cellIJK );
+
```

Comment:
Maybe better to use `std::optional<std::array<size_t, 3>> `instead of `std::string` here? And add a `cellIjkString` which converts?

Reply:
Changed to optional<caf::VecIjk>